agent.py

```diff
@@ -85,26 +90,28 @@ def __init__(self,
                                 )
 
         # np wizardry
-        self.r_fov = 150
         self.do_x, self.do_y = np.indices((int(window_dimensions[0] /
```

Comment:
refactor this shit and fix pep

Reply:
![afbeelding](https://github.com/Finnyy012/TUPOLEVGAMING/assets/94832237/99210a7e-eb2d-4c3a-931a-03fa22821219)